sdk/src/main/java/com/bugsnag/android/AppData.java

```diff
@@ -1,12 +1,15 @@
 package com.bugsnag.android;
 
+import android.app.ActivityManager;
 import android.content.Context;
 import android.content.pm.ApplicationInfo;
 import android.content.pm.PackageManager;
+import android.os.SystemClock;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
 import java.io.IOException;
+import java.util.List;
 
 /**
  * Information about the running Android app which doesn't change over time,
```

Comment:
Does this comment need updating now some of the fields have been extracted into another class?